docs/_v1_0/2_signeringsoppdrag.md

```diff
@@ -44,6 +44,7 @@ Ved opprettelse av signeringsoppdrag kan følgende felter oppgis:
 | Signaturtype            | Valgfritt        | Valgfritt         | se [signaturtype](#signaturtype) |
 | Sikkerhetsnivå          | Valgfritt        | Valgfritt         | se [sikkerhetsnivå](#sikkerhetsniv) |
 | Melding til mottaker(e) | Valgfritt        | Valgfritt         |   |
+| Undertegners identifikator | Valgfritt     | Valgfritt         | se [undertegners identifikator](#undertegners-identifikator) |
```

Comment:
Thats right, gadd ikke korrigere markdownen for at alle radene skulle være «riktig formattert» bare fordi det var noen tegn for mye på denne raden. Deal with it 😎